src/chat.rs

```diff
@@ -874,8 +874,9 @@ pub fn send_text_msg(
 ) -> Result<MsgId, Error> {
     ensure!(
         chat_id > DC_CHAT_ID_LAST_SPECIAL,
-        "bad chat_id = {} <= 9",
```

Comment:
i think this is just the error messages?
so the printed output would be `bad chat_id = 8 <= 9`?
maybe it is better to have DC_CHAT_ID_LAST_SPECIAL in the error message literally, sth as 
ensure!(
         chat_id > DC_CHAT_ID_LAST_SPECIAL,
        "chat#{} must be larger than DC_CHAT_ID_LAST_SPECIAL", chat_id);